src/trustBadge/Capsule/index.tsx

```diff
@@ -48,8 +48,8 @@ const styles = StyleSheet.create({
   },
   chevron: {
     color: '#0A3942',
-    fontSize: 26,
-    marginTop: -4,
+    ...(Platform.OS === 'ios' && { fontSize: 24 }),
+    marginTop: Platform.OS === 'ios' ? -4 : -8,
```

Comment:
[question]

Where is the font size defined for Android? If it is a global setting, can app developers alter it in their application so the font will end up larger than we assumed for the chevron :) For this reason, should we explicitly set font size for Android?

Reply:
> Where is the font size defined for Android?

Good question, I don't know. As the code stands currently, I don't think we're explicitly setting a font size for Android, so I assume it will just be whatever the default is (maybe 16px / 1rem?)

But yeah, I think it's a good point. I just did some more testing and found that `fontSize: 14` seems to be working well for Android, so I set it to that, in 5ef909a.